src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           _RunOnline: false
         Ubuntu1804-Offline:
```

Comment:
The leg names should also be updated otherwise it will be terribly confusing in the AzDO UI.